pkg/instancetype/instancetype_test.go

```diff
@@ -2366,7 +2369,7 @@ var _ = Describe("Instancetype and Preferences", func() {
 						},
 					},
 				},
-				nil,
+				&v1.VirtualMachineInstanceSpec{},
```

Comment:
Nil specs not allowed anymore?

Reply:
See https://github.com/kubevirt/kubevirt/pull/13036#issuecomment-2405629143  - for whatever reason this leads to a panic on s390x but not on amd64. `nil` is actually an invalid test case anyway for this code path.

Reply:
ACK